js/setup.js

```diff
@@ -0,0 +1,64 @@
+var names = ['Иван', 'Хуан' + 'Себастьян', 'Мария', 'Кристоф', 'Виктор', 'Юлия', 'Люпита', 'Вашингтон'];
+var surnames = ['да' + 'Марья', 'Верон', 'Мирабелла', 'Вальц', 'Онопко', 'Топольницкая', 'Нионго', 'Ирвинг'];
+var coatColors = ['rgb(101, 137, 164)', 'rgb(241, 43, 107)', 'rgb(146, 100, 161)', 'rgb(56, 159, 117)', 'rgb(215, 210, 55)', 'rgb(0, 0, 0)'];
+var eyesColors = ['black', 'red', 'blue', 'yellow', 'green'];
+
+var setupSelector = document.querySelectorAll('.setup');
+setupSelector.classList.remove('hidden');
+document.querySelector('.setup-similar').classList.remove('hidden');
+
+var similarListElement = document.querySelector('.setup-similar-list');
+var similarWizardTemplate = document.querySelector('#similar-wizard-template')
+    .content
+    .querySelector('.setup-similar-item');
+
+var randomArrayValue = function(myArray) {
+  var rand = myArray[Math.floor(Math.random() * myArray.length)];
+  return rand;
```

Comment:
```suggestion
  return myArray[Math.floor(Math.random() * myArray.length)];
```
1. можно в одну строчку, без объявления переменной
2. в названии функции нужно добавить глагол. Например, getRandomArrayValue